site/docs/uk/guide/reactions.md

```diff
@@ -0,0 +1,171 @@
+# Реакції
+
+Боти можуть працювати з реакціями на повідомлення.
+Існує два типи реакцій: звичайні емодзі та користувацькі емодзі.
+
+## Реагування на повідомлення
+
+Боти можуть додавати одну емодзі-реакцію на повідомлення.
+
+У тих же випадках боти також можуть реагувати на повідомлення користувацькими емодзі, хоч боти і не можуть мати [Telegram Premium](https://telegram.org/faq?setln=uk).
```

Comment:
```suggestion
У тих же випадках боти також можуть реагувати на повідомлення користувацькими емодзі, хоч боти і не можуть мати [Telegram Premium](https://telegram.org/faq_premium?setln=uk#q-can-i-buy-a-premium-subscription-for-my-bots).
```

Reply:
I think this change will also need to be made in the English version, and then in the translations. But this will have to be done after the merge of this PR.
@KnorpelSenf

Reply:
In the meantime, let's fix this little mistake
```suggestion
У тих же випадках боти також можуть реагувати на повідомлення користувацькими емодзі, хоч боти і не можуть мати [Telegram Premium](https://telegram.org/faq_premium?setln=uk).
```

Reply:
The link was intended for people who do not know what Telegram premium is. There are people who are not very aware of the feature set of Telegram, so I tend to link to important pages just to be sure.

We may want to include a second link to the question you're bringing up, but this would indeed be a subsequent PQ.

Reply:
I think it's in this sentence that we should add a link to the specific question, and below it add a note that if a user hears about premium for the first time, he should read the entire page.